public/configs/v1/env.json

```diff
@@ -315,6 +316,39 @@
             "isSlTpLimitOrdersEnabled": false
          }
       },
+      "dydxprotocol-dev-3": {
+         "name": "v4 Dev 3",
+         "ethereumChainId": "11155111",
+         "dydxChainId": "dydxprotocol-testnet",
+         "chainName": "dYdX Chain",
+         "chainLogo": "/dydx-chain.png",
+         "squidIntegratorId": "dYdX-api",
+         "isMainNet": false,
+         "endpoints": {
+            "indexers": [
+               {
+                  "api": "http://dev3-indexer-apne1-lb-public-1613790025.ap-northeast-1.elb.amazonaws.com",
```

Comment:
[q] why are the urls using aws urls? looks like other dev envs have namespaced urls (i'm assuming via route 53?)

Reply:
Will update when new urls are provisioned.